### PR TITLE
daemon, client, cmd/snap: add install_date to rest api, client, and snap list output

### PR DESCRIPTION
**Pull request by chipaca**
Dates, dates, every where,
And all the boards did shrink;

### CODE REVIEW

File: daemon/snap.go [hunk 1]
```diff
@@ -133,7 +133,7 @@ func mapSnap(localSnaps []*snappy.Snap, remotePart *snappy.RemoteSnap) map[strin
 		// *) not the actual right rollback because we aren't
 		// marking things failed etc etc etc)
 		//
-		if len(localSnaps) > 1 {
+		if len(localSnaps) == 2 {
```
Comment (mvo5) [inline daemon/snap.go]: This looks unreleated?

Comment (chipaca) [inline daemon/snap.go]: yes. It prevents a panic tho :-)

**Comment by mvo5** [comment]
Nice and straightforward, just one question about a line that looks unrelated but if its added on purpose this is all fine.

**Comment by mvo5** [comment]
:+1: (needs conflicts resolving)

**Comment by niemeyer** [comment]
Can we hide the column behind a flag?  This feels like a huge column on everybody's screen that doesn't feel so critical.  Nevertheless, feels nice to have it available in the API and behind a flag.

**Comment by chipaca** [comment]
@niemeyer yes we can. All things considered I'd rather leave it for later though.

File: cmd/snap/cmd_list.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	for k := range snaps {
+		names[i] = k
+		i++
+	}
+	sort.Strings(names)
+
+	w := tabwriter.NewWriter(Stdout, 5, 3, 1, ' ', 0)
+	defer w.Flush()
+
+	fmt.Fprintln(w, i18n.G("Name\tDate\tVersion\tDeveloper"))
```
Comment (niemeyer) [inline cmd/snap/cmd_list.go]: To move this in today, can we please drop the Date and Developer columns until we figure details?

The installation date I really don't think it's so relevant to have here, and the developer we agreed to not display while developer == publisher, which is the only case we have right now. So for the time being we can just list a very simple "Name Version" output, and Revision once we have that.

We should definitely keep everything else about the date in this branch, though, so it's easy to add it or do something else once we find out what that is.

**Comment by niemeyer** [comment]
LGTM

**Pull request merged**